docs/modules/ROOT/pages/adr/0031-naming-scheme-for-servala-cluster-names-and-urls.adoc

```diff
@@ -0,0 +1,75 @@
+= ADR 0031 - Naming scheme for servala cluster names and URLs
+:adr_author:    Nicolas Bigler
+:adr_owner:     Schedar
+:adr_reviewers: 
+:adr_date:      2025-04-11
+:adr_upd_date:  2025-04-11
+:adr_status:    draft
+:adr_tags:      servala
+
+include::partial$adr-meta.adoc[]
+
+[NOTE]
+.Summary
+====
+With the launch of Servala we will have multiple clusters running on different cloud providers and on different regions. To ensure consistency and ease of management, it is important to establish a naming scheme for these clusters. This ADR outlines the proposed naming scheme for servala cluster names and URLs.
+====
+
+== Context
+
+=== Cluster names
+
+We have two different kind of clusters for servala:
+* Control-plane clusters: Those run crossplane and contain all the crossplane resources like functions, composites, api-server etc.
+* Service clusters: Those run the workloads that are managed by crossplane.
+
+To properly distinguish and identify each cluster, the cluster name should contain the following information:
+* The CSP where the cluster is running: Cloudscale, Exoscale, etc.
+* The region where the cluster is running: `ch-gva-2`, `lpg1`, etc.
+* The type of cluster: `control` or `service`
+* The stage of the cluster: `test`, `prod`
+* A counter for each cluster to ensure uniqueness.
+
+Lieutenant already gives us part of the naming scheme. Each cluster should be prefixed with `c-` followed by the tenant, so that it can easily identify which tenant owns the cluster.
+
+=== URLs
+
+We have to differnt URLs that are important:
+* The clusters API endpoint
+* The apps endpoint of Openshift for applications on the service clusters
```

Comment:
```suggestion
We have two different URLs that are important:
* The Kubernetes API endpoint
* The apps endpoint of OpenShift for applications on the service clusters
```

I don't fully understand the "apps" endpoint because for Servala this one doesn't really matter. This mainly is relevant for a full-fledged OpenShift cluster to have a default domain for apps running on them.